pkg/extensions/search/resolver.go

```diff
@@ -575,6 +577,8 @@ func BuildImageInfo(repo string, tag string, manifestDigest godigest.Digest,
 		lastUpdated = imageConfig.History[len(imageConfig.History)-1].Created
 	}
 
+	manifestAnnotations := common.GetAnnotations(manifest.Annotations, manifest.Config.Annotations)
```

Comment:
Reuse the annotations object defined above. manifest,Config will not contain annotations anyway.